ppdet/data/transform/operators.py

```diff
@@ -1672,3 +1730,69 @@ def __call__(self, sample, context=None):
         bbox[:, :2] = bbox[:, :2] + bbox[:, 2:4] / 2.
         sample['gt_bbox'] = bbox
         return sample
+
+
+@register_op
```

Comment:
is it necessary to merge into master?

Reply:
This visualized code is helpful, can merge